doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -997,9 +997,17 @@ Generation of induction principles with ``Scheme``
   :n:`Minimality for @reference Sort @sort_family`
     Defines a non-dependent elimination principle more natural for inductively defined relations.
 
+  :n:`Boolean Equality for @reference`
+     Tries to generate a Boolean equality.
+     If :token:`reference` involves independent constants or other
+     inductive types, it is recommended to define their equality
+     first.
+
   :n:`Equality for @reference`
      Tries to generate a Boolean equality and a proof of the decidability of the usual equality.
```

Comment:
```suggestion
  :n:`{? Boolean } Equality for @reference`
     Tries to generate a Boolean equality.  If :n:`Boolean` is not specified, the tactic also
     generates a proof of the decidability of the usual equality.
```

Reply:
"the usual equality" is vague, it that easy to clarify (if you're willing)?

Reply:
I wrote instead "propositional equality". I also extended a bit the description.